Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+1. Select ‘Create a new branch for this commit’
+1. Create a name for the new branch that explains **what it is** 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit2.png" />. 
+1. Click Propose Changes. 
+1. Click on the Pull Requests tab. Do NOT click the green ‘Create Pull Request’ button that appears at the bottom of the next screen.
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit7.png" />
+
+## 4. Submit a Pull Request (Submit edits for review)
+
+1. Click ‘New Pull Request’. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR1.png" />
+1. Make sure that your base repository is vmware-tanzu/tanzu-dev-po..., and the head repository is your account.  
+
+1. Change the dropdown for your “compare: main” to the name of the branch you have created. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR2.png" />
+1. Scroll down to check that text highlighted in green is the change you want to make.   
+1. Click Create Pull Request
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR5.png" />
+1. Click Create Pull Request again. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR6.png" />
+1. If you scroll down, you can see that the edits are going through some automated checks. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR7.png" />
+1. If you want to, you can keep the screen running until all the tests have passed, just to have a peace of mind. In the list of checks, some people can see a row below WIP called netlify. If you see it, you can click on the netlify row after all the checks have completed to preview how it will look on the site.
+1. Otherwise, feel free to close GitHub and continue with your life. The checks will continue running even if you leave the page. Once the change passes through all the checks, it will land on the reviewers’s plate, and all you have to do is wait for it to be reviewed, approved and merged. 
+
+1. You’ve successfully suggested a change to the practices site! 
+
+
+> ## Short FAQ
+>
+> ### How will I know if the changes have passed the checks, been rejected, approved or merged?
```

Comment:
Is this whole section intentionally prefixed with `>`?

The `>`  is typically used for quotes, which doesn't really make sense here. I would remove it.